alert/models.py

```diff
@@ -90,14 +114,16 @@ def resub_url(self):
         return '{}/s/{}'.format(settings.PCA_URL, url.short_id)
 
     def alert(self, forced=False, sent_by=''):
-        if forced or not self.notification_sent:
+        if forced or not (self.notification_sent or self.muted or self.deleted):
             text_result = Text(self).send_alert()
```

Comment:
The send_alert() method is going to need to be changed as well, to look at the `UserData` before looking at `Registration.phone` and `Registration.email`. I believe this is in `alert/alert.py`